src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3910,8 +3910,14 @@
   <data name="Arg_EntryPointNotFoundExceptionParameterizedNoLibrary" xml:space="preserve">
     <value>Unable to find an entry point named '{0}' in native library.</value>
   </data>
-  <data name="Arg_DllNotFoundExceptionParameterized" xml:space="preserve">
-    <value>Unable to load native library '{0}' or one of its dependencies.</value>
+  <data name="DllNotFound_Windows" xml:space="preserve">
+    <value>Unable to load DLL '{0}' or one of its dependencies: {1}</value>
```

Comment:
Since {1} seems likely to be a sentence, I think it reads better (and perhaps it's more conventional) to separate with a period instead of a colon. For example we do this in the console output for FailFast. Etc

Reply:
This is the existing string from mscorrc: https://github.com/dotnet/runtime/blob/abca8a1d247350467af66f76b7c15a47d5dcc4b1/src/coreclr/dlls/mscorrc/mscorrc.rc#L165 that dates back to .NET Framework. I'll keep it for consistency.